tests/array/first.js

```diff
@@ -0,0 +1,15 @@
+var chai = require('chai');
+var expect = chai.expect;
+
+require('../../array/first');
+
+describe('first', function() {
+  it('should grab the first item from the array', function() {
```

Comment:
says it should grab the first item from an array, but you're testing more than that. Also, make the test array a variable instead of typing it so much